### ISSUE TITLE
Add MB/GB option to Memory

### ISSUE THREAD

**Issue by caseylargent** [unlhcc/HCCGo#164]

**Comment by caseylargent** [comment]
Did we talk about this being a design philosophy for HCCGo? Or would it be just as easy to convert the units based on what is passed in?

**Comment by djw8605** [comment]
Nah, this isn't a design philosophy.  I'm not sure on the best way to do this though.
